index.html

```diff
@@ -31,21 +31,22 @@
         <div class="container-nav">
           <a href="index.html" class="logo-title">Marlyn Mayienga</a>
           <nav class="nav-bar">
-            <ul>
+            <ul class="menu">
```

Comment:
- [ ] Kindly ensure the mobile menu option when clicked, take the user to that section of the page eg contact should take the user to contact section.
- [ ] It would also be nice making the background a bit blurred the user shouldn't be able to read the text in the background.
- [ ] Kindly check you mobile view the logo is missing 
![image](https://user-images.githubusercontent.com/61408860/180021462-6da0220d-93ac-4bd6-b361-39d320215a96.png)